### DISCUSSION TITLE
Hearing Patch affected something

### DISCUSSION THREAD

**Discussion by Dexteriority** [winstxnhdw/lc-hax#390] [Q&A] [answered]
I dont take it as a glitch/bug but i think it might be just how the game works...
So basically with **/hear** you cant use Walkie-Talkie, the walkie will just glitched in the inventory. (can't be held)

I think of it as how the game works because _you_ hearing everyone is already functioned as a walkie talkie right?
So this it is normal right? turning off the **/hear** will give you access back to hold the walkie-talkie

**Comment by winstxnhdw** [comment, marked as answer]
I hate the `hear` command because it has so many extraneous side-effects like this, and I have not been able to find out why.

**Reply by Dexteriority** [reply]
Probably the game confused with what to do, I mean this one is not that bad. its fine!
